src/pages/CreateEvent.js

```diff
@@ -0,0 +1,9 @@
+import React from "react";
+
+export default function CreateEvent() {
```

Comment:
페이지 컴포넌트는 끝에 어미로 Page가 들어가면 구분하기 편할 것 같음
```suggestion
export default function CreateEventPage() {
```